src/Form/Constraint/VimeoValidator.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Message\Cog\Form\Constraint;
```

Comment:
`Constrants` (plural) is used below, but `Constraint` is used here. Consistency?